specs/typography/typography.md

```diff
@@ -0,0 +1,143 @@
+# Typography
+
+*TODO: Sections that will be removed upon review are marked with ~~striketrough~~, remaining for commenting purposes.*
+
+## Overview
+
+*Typography* defines the underlying type size system for applying to elements, css classes, or within components. This spec does not propose any components of its own.
+
+### Background
+
+A type ramp can help create balance and contrast in formatted content. Sizes from a ramp can be applied linearly to form progression and information hierarchy, or selected artistically.
+
+The use of type ramps has existed since the early days of printing, where fixed sizes and intervals were common. In digital representation we have increased flexibility and options, including calculating more precise and variable scales.
+
+This typography system provides an extensible structure anchored around a base size, from fixed values or generated using a modular scale.
+
+### Use Cases
+
+- Formatting main body content
+- Defining page hierarchy using headings
+- Providing a label for an input field
+- Adding a caption to an image or figure
+
+### Features
+
+- An extensible type ramp around a base font size
+- CSS variables to apply the type ramp
+
+### Risks and Challenges
+
+Further typography-related specs build on the underlying principles and resources defined here. It will be important to provide meaningful ways to allow for semantic styles and consistent application of appearances in balance with flexibility and maintenance cost.
+
+### Prior Art / Examples
+
+- [FAST-DNA (React)](https://explore.fast.design/components/typography)
+- [Material Design](https://material.io/design/typography/the-type-system.html)
+- [Bootstrap](https://getbootstrap.com/docs/4.4/content/typography)
+- [Ant Design](https://ant.design/components/typography)
+- [Mineral UI](https://mineral-ui.netlify.com/typography)
+
+---
+
+## Design
+
+The default type ramp uses a selection of values from [Microsoft Fluent Design](https://www.microsoft.com/design/fluent).
```

Comment:
I'm not seeing values after the jump here and I end up down a rabbit trail trying to find them. Do we need the link here or will the image do?

Reply:
Good point. Link removed.